New site: Rough home draft (#2244)
meilisearch/documentation: home.mdx

description: Here you'll find everything you need to learn and create with our lightning fast search engine.
---

# Documentation

> Explore our guides and references to learn how to use Meilisearch in your projects.

Reviewer: ```suggestion
Learn to use Meilisearch in your projects by exploring our guides and API reference.
```

<Capsule>
  ## Quick start

>   New here? Check out the [Quick start guide](/learn/getting_started/quick_start.mdx) and learn how to launch a basic Meilisearch instance.

Reviewer: I feel pretty confident that this will fail since I don't think inserting Markdown into JSX is supported but let's try it as a test 😎

Author: I'm pretty sure this works. Looking at files converted by basement, many use e.g. bold inside `<Capsule>`s:  https://meilisearch-staging.vercel.app/docs/learn/advanced/asynchronous_operations#task-queue (Note right above task queue, for example).

My understanding is that markdown is fine inside components, but fails inside props.

🟢 
```jsx
<Component>
  _Oxford comma_
</Component>
```

❌ 
```jsx
<Component propertyName="_Oxford comma_"/>
```

Reviewer: Makes sense, I think you're right! Time to find out for sure 😎 🚀

</Capsule>

<h2 className="text-5xl">Explore The docs</h2>

<Featured items={[
  {
    icon: 'cube',
    content: 'Consult the Meilisearch [API reference](/reference/api).'
  },
  {
    content: 'Looking for SDK documentation? Check out this list of [official Meilisearch libraries](/learn/what_is_meilisearch/sdks.mdx).',
    icon: 'compass'
  },
  {
    content:

>       'Announcing [Meilisearch Cloud](https://cloud.meilisearch.com/register): the best way to add Meilisearch to your project',

Reviewer: ```suggestion
      '[Meilisearch Cloud](https://cloud.meilisearch.com/register): the easiest way to add Meilisearch to your project',
```

Author: I thought we were trying to avoid words like "easy" and "simple"?

Reviewer: We do avoid those words in a documentation context, but I don't think we ever decided to avoid them in a marketing context.

Reviewer: Either way, this suggestion is pretty subjective so I'm fine leaving it as is.

    icon: 'cloud'
  },
  {
    content:

>       'Read the Meilisearch blog for more tips, tutorials, and information about our company.',

Reviewer: ```suggestion
      'For more tips, tutorials, and information about our company, take a look at the Meilisearch blog.',
```
